resources/assets/less/bem/osu-page-header.less

```diff
@@ -42,6 +42,35 @@
     .at2x-simple("/images/headers/tournament.jpg");
   }
 
+  &--mp-history {
+    .at2x-simple("/images/headers/mp-history.png");
+    background-position: center;
+    height: 125px;
+  }
+
+  &--contests {
```

Comment:
actually i have no idea, something must have messed up during the merge i guess, as this [is still attributed to nekodex](https://puu.sh/qFG3j.png)